app/views/users/_form.html.erb

```diff
@@ -0,0 +1,26 @@
+ <%= form_for(@user) do |f| %>
+  <%= render "shared/error_messages", object: @user %>
+
+  <%= f.label t(".name") %>
+  <%= f.text_field :name, class: "form-control" %>
+
+  <%= f.label t(".email") %>
+  <%= f.email_field :email, class: "form-control" %>
+
+  <%= f.label t(".password") %>
+  <%= f.password_field :password, class: "form-control" %>
+
+  <%= f.label t(".confirmation") %>
+  <%= f.password_field :password_confirmation, class: "form-control" %>
+  
+  <%= f.label t(".role")%><br />
+
+  <%= f.radio_button :role, :trainee, checked: true %>
```

Comment:
https://stackoverflow.com/questions/45374174/how-to-get-key-name-from-a-enum-value-in-rails/45374287
https://stackoverflow.com/questions/38668069/rails-5-get-enum-key-via-integer-value

tham khảo 2 link trên 
-> `:trainee` thành `User.roles.key(User.roles[:trainee])`